massdriver-application-aws-ecs-service/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "md_metadata" {
```

Comment:
<3 
```
variable "name" {
  type = string
}

variable "tags" {
  type = any
}
```

Reply:
This is only the case for non `massdriver-*` modules. Since this is a massdriver module its okay for us build it with massdriver assumptions built-in - e.g. to assume `md_metadata` and other massdriver-specific variables/artifact-definitions will come in.

If this was an AWS RDS module, we don't want to **assume** its running in Massdriver and thus don't want to force any massdriver-specific artifacts or variables to be there.